portfolio/src/main/webapp/index.html

```diff
@@ -7,7 +7,7 @@
   <link rel="stylesheet" href="style.css">
   <script src="script.js"></script>
 </head>
-<body onload="loadComments()">
+<body onload="loadComments(); checkLogin();">
```

Comment:
Nit: Provide an onload function whose name doesn't need to change when you want more functionality?

```html
<body onload="onIndexLoad()">
```

```js
function onIndexLoad() {
  loadComments();
  checkLogin();
}
```